spec/recursive_open_struct_spec.rb

```diff
@@ -42,6 +42,11 @@
         it { subject.methods.map(&:to_sym).should include :blah= }
         it { subject.methods.map(&:to_sym).should_not include :asdf }
         it { subject.methods.map(&:to_sym).should_not include :asdf= }
+
+        describe 'alias_methods' do
+          it { subject.method(:to_hash).should eq subject.method(:to_h) }
+          it { subject.method(:recurse_over_array).should eq subject.method(:array) }
```

Comment:
Same, the `recurse_over_array` method is not really meant to be part of the ROS's public API, although demonstrating that `to_hash` and `to_h` are the same is fine.